src/main/java/edu/kis/powp/jobs2d/command/Commands/OperateToCommand.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.command.Commands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.Interfaces.DriverCommand;
+
+public class OperateToCommand implements DriverCommand {
+
+    private final int x;
+    private final int y;
+    private final Job2dDriver driver;
+
+
+    public OperateToCommand(int x, int y, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).